internal/converter/internal/staticconvert/testdata/traces.river

```diff
@@ -165,3 +225,14 @@ otelcol.exporter.otlp "_1_0" {
 }
 
 otelcol.exporter.logging "_1_default" { }
+
+otelcol.connector.spanmetrics "_1_default" {
```

Comment:
load balancing in the config is why there are 2 trace pipelines in the otel config. I made sure that the spanmetrics gets applied to all pipelines so you see it on both in this test.